frame/collective/src/lib.rs

```diff
@@ -2158,11 +2251,39 @@ mod tests {
 	#[test]
 	#[should_panic(expected = "Members cannot contain duplicate accounts.")]
 	fn genesis_build_panics_with_duplicate_members() {
-		collective::GenesisConfig::<Test> {
+		pallet_collective::GenesisConfig::<Test> {
 			members: vec![1, 2, 3, 1],
 			phantom: Default::default(),
 		}
 		.build_storage()
 		.unwrap();
 	}
+
+	#[test]
+	fn migration_v4() {
+		new_test_ext().execute_with(|| {
+			use frame_support::traits::PalletInfo;
+
+			let old_pallet =
+				<Test as frame_system::Config>::PalletInfo::name::<Collective>().unwrap();
+			let new_pallet = "NewCollective";
+			crate::migrations::v4::pre_migrate::<Collective, _>(old_pallet, new_pallet);
+			crate::migrations::v4::migrate::<Test, Collective, _>(old_pallet, new_pallet);
+			crate::migrations::v4::post_migrate::<Collective, _>(old_pallet, new_pallet);
+
```

Comment:
I don't see any assertions here to make sure the prefix is actually `NewCollective` now